lisp/template.ros

```diff
@@ -0,0 +1,213 @@
+#!/bin/sh
+#|-*- mode:lisp -*-|#
+#| Register a file as a template.
+exec ros -Q -m roswell -L sbcl-bin -- $0 "$@"
+|#
+
+#|
+
+ros [options] template add [OPTIONS] TEMPLATE ARGS...
+
+* each argument is separated by --, instead of being surrounded by parentheses.
+* &key, &optional, &rest corresponds to --key, --optional, --rest, respectively.
+* Abbreviation as -k, -o, -r are available.
+
+|#
+
+#-ros.match
+(ros:include "match")
+#-ros.util
+(ros:include "util")
+
+#-ros.template
+(defpackage :ros.script.template
+  (:use :cl :ros.util :ros.match :alexandria))
+
+(in-package :ros.script.template)
+
+#-ros.template
```

Comment:
redefiniton warning is muffled completely. a49ce0f8ebce61bac380e103abb1858239be295b 

```
#+ros.package
(progn

)
```

just to prevent redefiniton warning is not required anymore :)